modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -0,0 +1,138 @@
+* You downloaded the xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent-retrieve_installing-with-agent-based-installer[Agent-Based Installer] and the command-line interface (CLI) from Red Hat’s Hybrid Cloud Console.
+* For a disconnected environment, you created a container image registry, such as Red Hat Quay. See xref:../../installing/disconnected_install/installing-mirroring-creating-registry.html#mirror-registry-introduction_installing-mirroring-creating-registry[Mirror registry for Red Hat OpenShift introduction].
+* You have logged into the {product-title} with administrator privileges. 
+
+.Procedure
+
+. Configure the `install-config.yaml` configuration file to meet the needs of your organization. 
++
+.Example `install-config.yaml` configuration file that demonstrates setting an external platform
++
+[source,yaml]
+----
+# install-config.yaml
+apiVersion: v1
+baseDomain: <base_domain> <1>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  network type: OVNKubernetes
+  machineNetwork:
+  - cidr: <ip_address_from_cidr> <2>
+  serviceNetwork: 
+  - 172.30.0.0/16
+compute:
+  - architecture: amd64 <3>
+  hyperthreading: Enabled
+  name: worker
+  replicas: 0
+controlPlane:
+  architecture: amd64 <3>
+  hyperthreading: Enabled
+  name: master
+  replicas: 3
+platform:
+   external:
+    platformName: oci <4>
+    cloudControllerManager: External
+sshKey: <public_ssh_key> <5>
+pullSecret: '<pull_secret>' <6>
+# ...
+----
+<1> The base domain of your cloud provider. 
+<2> The IP address from the VCN that the CIDR allocates to resources and components that operate on your network.
+<3> Depending on your infrastructure, you can select either `x86_64`, or `amd64`.
+<4> Set `OCI` as the external platform, so that {product-title} can integrate with {oci}. 
+<5> Specify your SSH public key.
+<6> The pull secret that you need for authenticate purposes when downloading container images for {product-title} components and services, such as Quay.io. See link:https://console.redhat.com/openshift/install/pull-secret[Install {product-title} 4] from the Red Hat Hybrid Cloud Console. 
+
+. Create a directory on your local system named `openshift`. 
++
+[IMPORTANT]
+====
+Do not move the `install-config.yaml` and `agent-config.yaml` configuration files to the `openshift` directory.
+====
+
+. From the link:https://github.com/oracle-quickstart/oci-openshift[`oracle-quickstart / oci-openshift`] GitHub web page, select the **<> Code** button and click **Download ZIP**. Save the archive file to your `openshift` directory,  so that all the CCM and CSI manifests exist in the same directory. The downloaded archive file includes files for creating cluster resources and custom manifests. 
+
```

Comment:
Just wanting to note the CSI being referred here is the Oracle CSI File Driver.  Many customers will be looking for a CSI Block Driver, and I want to make sure we're clear that this is the Oracle CSI File Driver.

CC @makentenza @racedo

Reply:
@dfitzmau please update `CCM and CSI manifests` to `Oracle CCM and Oracle CSI manifests`